wm-paper/rfc/index.md

```diff
@@ -0,0 +1,91 @@
+# Stable UI
+
+After a lot of learnings, we can be proud that we have figured out what Watermelon will look like for the short and medium term. We have a stable UI that we can use to build the extension.
+
+## What is the stable UI?
+
+For this, we thought of all the possible integrations that will come in the future, and the broad categories of information that we will need to display. We also thought about the different ways to display this information, and the different ways to interact with it.
+
+Here are some examples:
+
+- Code Storage
+  - GitHub
+  - GitLab
+  - BitBucket
+- Ticketing
+  - Jira
+  - Trello
+  - Asana
+  - GitHub Projects
+  - Linear
+  - Notion
+- Communication
+  - Slack
+  - Discord
+  - Teams
+  - Zoom
+  - Google Meet
+- Documentation
+  - Confluence
+  - Notion
+  - Google Docs
+  - Microsoft Docs
+  - GitHub Wiki
+- CRMs
+  - Hubspot
+  - Salesforce
+  - Pipedrive
+  - Zoho
+- Analytics
+  - Google Analytics
+  - Mixpanel
+  - Amplitude
+  - Segment
+- Bug Catching
```

Comment:
We can definitely index information from bug catching systems. Would need to discuss at depth what exactly.

Reply:
Let the users tell us